dev/io.openliberty.io.opentelemetry.1.1/src/io/opentelemetry/api/package-info.java

```diff
@@ -0,0 +1,21 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+/**
+ * @version 1.0
+ */
+@org.osgi.annotation.versioning.Version("1.0")
```

Comment:
Sorry, my previous comment on this line was misleading.

The version for this package is set in the bnd.bnd (to match the version of OTel) so we shouldn't also have a version annotation here.